src/simulation/monsters/special/Hespori.ts

```diff
@@ -0,0 +1,60 @@
+import LootTable from '../../../structures/LootTable';
+import Loot from '../../../structures/Loot';
+import { ItemBank } from '../../../meta/types';
+import { roll } from '../../../util/util';
+import Monster from '../../../structures/Monster';
+
+const HesporiTable = new LootTable()
+	.add('Attas seed', [1, 2], 1)
+	.add('Iasor seed', [1, 2], 1)
+	.add('Kronos seed', [1, 2], 1);
+
+const OtherTable = new LootTable()
+	.add('Watermelon seed', [10, 20], 2)
+	.add('Snape grass seed', [6, 16], 2)
+	.add('White lily seed', [8, 18], 5)
```

Comment:
can you add new lines between the different groups in the same way the wiki has a new table for each type, makes it a little easier to read/compare imo